Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM centos/ruby-23-centos7
+FROM centos/ruby-24-centos7
 
-ENV RUBY_VERSION="2.3.7" \
+ENV RUBY_VERSION="2.4.5" \
```

Comment:
I wonder if we need this ... it should already be set in the base image no?

Reply:
Yes no need https://github.com/sclorg/s2i-ruby-container/blob/07442541f87d6c82db0e2829fb07e69abc30af3b/2.4/Dockerfile#L8